docs/src/language_servers.md

```diff
@@ -760,21 +760,35 @@ See [Javascript/TypeScript](#javascripttypescript).
 ## Typst
 
 1. Install the [Typst](https://packagecontrol.io/packages/Typst) package from Package Control for syntax highlighting.
-2. Download the [typst-lsp](https://github.com/nvarner/typst-lsp/releases) language server executable for your platform.
+2. There are 2 available languages servers, and their configuration is mostly the same:
+    1. [tinymist](https://github.com/Myriad-Dreamin/tinymist) - has more features, like go to definition, rename, etc.
+    2. [typst-lsp](https://github.com/nvarner/typst-lsp/releases).
 3. Open `Preferences > Package Settings > LSP > Settings` and add the `"typst-lsp"` client configuration to the `"clients"`:
 
     ```jsonc
     {
         "clients": {
             "typst-lsp": {
                 "enabled": true,
-                "command": ["C:\\path\\to\\typst-lsp-win32-x64.exe"],  // adjust this path according to your platform/setup
+                "command": ["path/to/typst-lsp"],  // adjust this path according to your platform/setup
                 "selector": "text.typst"
             }
         }
     }
     ```
 
+    If you want to provide some initialization options to tinymist, like exportPdf of typstExtraArgs, set it like
+
+    ```jsonc
+    "tinymist": {
+        // ...
+        "initializationOptions": {
+            "exportPdf": "never",
+            "typstExtraArgs": [],
+        }
+    }
```

Comment:
If there exist two servers then this section should be refactored to have the `## Typst` header and two separate `### ...` sub headers for each server.

Reply:
fixed